src/v3/src/components/Form/renderers.tsx

```diff
@@ -133,6 +133,26 @@ export default [
     tester: ({ type }) => type === 'Divider',
     renderer: Divider,
   },
+  {
+    tester: ({
+      options: {
+        inputMeta: { options } = {} as Input,
+        format,
+        customOptions,
+      },
+    }: FieldElement) => (Array.isArray(customOptions) || Array.isArray(options)) && format === 'radio',
+    renderer: Radio,
+  },
+  {
+    tester: ({
+      options: {
+        inputMeta: { options } = {} as Input,
+        format,
+        customOptions,
+      },
+    }: FieldElement) => (Array.isArray(customOptions) || Array.isArray(options)) && format === 'select',
+    renderer: Select,
+  },
```

Comment:
Had to move this up to ensure select/radio elements are rendered (the format option triggers these elements to be rendered).